drivers/mmcsd/Kconfig

```diff
@@ -130,6 +130,10 @@ config MMCSD_SDIO
 
 if MMCSD_SDIO
 
+config SDIO_XFRDEBUG
```

Comment:
but the common code doesn't use this symbol?

Reply:
removed this